# Size of the workspace in cm
geometry_msgs/Point size

# Position of the workspace's center relative to the robot frame
geometry_msgs/Point center_position
geometry_msgs/Point corner_position

Height_Map_Row[] height_map
Packing_info[] parcels
